wesnoth-AitF/cs.po

```diff
@@ -1,18 +1,15 @@
-# Matěj Žejdl, 2024
-#
 msgid ""
 msgstr ""
-"Project-Id-Version: Battle for Wesnoth\n"
+"Project-Id-Version: \n"
```

Comment:
Tohle by nemělo být smazané. Můj Poedit to nedělá, právě jsem to vyzkoušel. Nepoužil jsi jinou verzi než tu z mého repozitáře, větve main?